components/centraldashboard/README.md

```diff
@@ -45,8 +45,8 @@ Make sure you have the latest LTS version of `node` installed along with `npm`.
       from the front-end starting with `/api` are proxied to the Express
       server. All other requests are handled by the front-end server which
       mirrors the production configuration.
-4. To access the Kubenetes REST API, run `kubectl proxy --port=8083`.
-   - This runs [kubectl proxy](https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#proxy) to create a reverse proxy at http://localhost:8083. Requests from the front-end starting with `/jupyter`, `/metadata`, `/notebook` and `/pipeline` are proxied to the [Kubenetes API server](https://kubernetes.io/docs/tasks/administer-cluster/access-cluster-api/). See the [webpack config file](https://github.com/kubeflow/kubeflow/blob/master/components/centraldashboard/webpack.config.js) for more details.
+4. To access a Kubenetes service, run `kubectl port-forward -n kubeflow svc/<service-name> <service-proxy-port>:<service-port>` e.g. `kubectl port-forward -n kubeflow svc/jupyter-web-app-service 8085:80`.
```

Comment:
Could you instead have a list for this numbered step and expose each `kubectl port-forward` command that a user needs to run?

We can just have commands for the Services that we proxy in the webpack config

Reply:
Pushed a new commit for both of the comments @kimwnasptd.